pkgs/os-specific/linux/plymouth/default.nix

```diff
@@ -1,64 +1,119 @@
-{ stdenv, fetchurl, autoreconfHook, pkg-config, libxslt, docbook_xsl
-, gtk3, udev, systemd, lib
+{ lib
+, stdenv
+, fetchpatch
+, fetchFromGitLab
+, pkg-config
+, autoreconfHook
+, libxslt
+, docbook_xsl
+, intltool
+, gtk3
+, systemd
+, freetype
+, pango
+, cairo
+, libdrm
 }:
 
 stdenv.mkDerivation rec {
   pname = "plymouth";
-  version = "0.9.4";
+  version = "2020-12-07";
```

Comment:
If we use the `git describe` format, `version = "0.9.5-29-gc4ced2a"`, then the version numbers will remain monotonic if we switch back to a release. This format also works with `fetchFromGit*`, so we can use `rev = version` below.

Reply:
Wait... Can we maybe open an issue for that? U probably know about that annoying rule we have with "pname = "something-unstable" and `verison = YYY-MM-DD`. IIRC this is what they do in debian world? Anyways, I think it's worth looking into that but I think it's out of scope of this PR, and I will use what the manual has currently.

Reply:
But you haven’t changed `pname` to `"plymouth-unstable"`, so the monotonicity problem is in scope for this PR.

Reply:
> Anyways, I think it's worth looking into that but I think it's out of scope of this PR, and I will use what the manual has currently.

The manual is inaccurate because it just mentions name and not pname or version. Changing pname is not an option because it breaks many package definitions and repology.